### DISCUSSION TITLE
iFrame `frameborder` attribute is deprecated in HTLM5

### DISCUSSION THREAD

**Discussion by jimblue** [mpratt/Embera#106] [Ideas]
Hi,

First thank for your amazing lib :)

After running a W3C test on the Embera HTML output , I've noticed that the `frameborder` attribute was deprecated and should be removed: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#frameborder

> The value 1 (the default) draws a border around this frame. The value 0 removes the [border](https://developer.mozilla.org/en-US/docs/Web/CSS/border) around this frame, but you should instead use the CSS property border to control <iframe> borders.

For now I'm adding a filter to fix that:

```php
$embera->addFilter(static function ($response) use ($loading) {
            if (!empty($response['html'])) {
                $response['html'] = str_replace('frameborder="0"', "", $response['html']);
            }

            return $response;
});
``` 

But it would be to release a new Embera for everyone :)

**Comment by mpratt** [comment]
Hi @jimblue,
With embera I'm trying mostly not to modify the original response from the provider. If the response from the provider is not html complaint, there is not much that we can do about it other than using a filters to modify the response.

We could think about introducing a config option `[ 'html_complaint' => true]`  but this also introduces the need to check for all the responses and conditions that are not complaint and fix the output. Honestly, I think that could translate in a lot of work and I dont think its going to be used that much.

I'll be thinking about this when releasing a newer major version but for now I'll be moving this to a "discussion" since its not really an Issue.
